docs/ibis-for-pandas-users.ipynb

```diff
@@ -46,7 +46,7 @@
    "id": "afd136cd-18b5-48a2-8359-c47cde181c7b",
```

Comment:
<p>Can you remove this bit? <code>offset</code> requires a caveat because of ordering. This is valid in pandas, but not in DuckDB or any other SQL backend. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6048/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>